### PR TITLE
Fields mapping

### PR DESCRIPTION
**Pull request by ErnestaP**

### CODE REVIEW

File: harvester_docs.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# APS
+## Default running parameters
+1. APS harvester customize parameters:
+<table>
```
Comment (pamfilos) [inline harvester_docs.md]: please update tables with the markdown syntax ( | --- | ----| ) for better compatibility.. lets keep the html tags to as minimal as possible

File: springer_fields_mapping.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+| **$schema**              | <a href="#enricher">enricher</a>:<a href="#68">68</a>                 |                |   |
+
+
+
+# [enricher](#enricher)
+|                     |               |                                                     |
+|---------------------|---------------|-----------------------------------------------------|
+| Reference           | Field         | Enricher                                            |
+| <p id="68">[68]</p> | schema        | <a href="_get_schema">_get_schema</a>               |
+| <p id="69">[69]</p> | arxiv_eprints | <a href="_get_arxiv_eprints">_get_arxiv_eprints</a> |
```
Comment (pamfilos) [inline springer_fields_mapping.md]: change `<p>` tag to `<a>`..e.g `<p id="68">[68]</p>` => `<a href="#68" id="68">[68]</a>`..this way you can also share by a click-able link

**Review by pamfilos — CHANGES_REQUESTED** [review]
please squash commits in the end

File: aps_fields_mapping.md [hunk 3]
```diff
@@ -0,0 +1,802 @@
+# APS data processing flow
+![aps_dags_process](./aps_dags_process.png)
+
+# Final fields
+
+<table>
```
Comment (drjova) [inline aps_fields_mapping.md]: Thanks @ErnestaP, could we change the table from html to markdown?

Comment (drjova) [reply aps_fields_mapping.md]: @ErnestaP the table is still HTML :)

**Comment by drjova** [comment]
@ErnestaP Thanks. There is a typo `dicumentation`

**Comment by drjova** [comment]
@ErnestaP could you please squash the commits?

**Pull request merged**